content/examples/guacamole/guacamole.docker-compose.md

```diff
@@ -0,0 +1,3 @@
+```yaml title="docker-compose.yaml"
+
```

Comment:
Is this file meant to be this empty?

Reply:
I forgot about that. I was going to move the examples into that directory, but we haven't been consistent about it. We can remove this.